### PR TITLE
Seperate constants

### LINKED ISSUE #143: Move runtime constants to a separate file
Currently constants, types (e.g Block) are in runtime/lib.rs

### PR DESCRIPTION
**Pull request by Satoshi-Kusumoto**
Move most of the basic constants and configurable constants into two separate files. Generally nothing should be changed.
Closes #143 

@wangminqi fee related things (ex. WeightToFee) should go to runtime/src/constants.rs

### CODE REVIEW

**Comment by wangminqi** [comment]
Get it. related code will be relocated after #188

File: primitives/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[package]
+name = 'primitives'
+version = '0.1.0'
+authors = ["Litentry Dev"]
+edition = '2018'
```
Comment (geastwood) [inline primitives/Cargo.toml:5]: What does this `2018` mean?

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -59,31 +66,9 @@ use xcm_builder::{
...
-pub type BlockNumber = u32;
-
 /// The address format for describing accounts.
 pub type Address = MultiAddress<AccountId, ()>;
 
-/// Block header type as expected by this runtime.
-pub type Header = generic::Header<BlockNumber, BlakeTwo256>;
-
 /// Block type as expected by this runtime.
 pub type Block = generic::Block<Header, UncheckedExtrinsic>;
```
Comment (Kailai-Wang) [inline runtime/src/lib.rs:73]: Will this `Block` conflict with what's defined in `primitives`, or can it be safely deleted?

File: primitives/Cargo.toml (hunk 1 above)
Comment (Satoshi-Kusumoto) [reply primitives/Cargo.toml:5]: Means Rust 2018 edition, the latest one

File: runtime/src/lib.rs (hunk 2 above)
Comment (Satoshi-Kusumoto) [reply runtime/src/lib.rs:73]: In primitives, it is opaque::Block, and if you look at it carefully, with `pub use sp_runtime::OpaqueExtrinsic as UncheckedExtrinsic;`, the `UncheckedExtrinsic` is not the same as the one in runtime/lib.rs. Therefore these are two different `block` (I come firmed this before making this PR)

**Review by Kailai-Wang — APPROVED** [review]

**Review by h4n0 — APPROVED** [review]

**Pull request merged**